src/border_agent/border_agent.cpp

```diff
@@ -163,6 +169,8 @@ BorderAgent::BorderAgent(otbr::Ncp::RcpHost &aHost, Mdns::Publisher &aPublisher)
     , mBaseServiceInstanceName(OTBR_MESHCOP_SERVICE_INSTANCE_NAME)
 {
     mHost.AddThreadStateChangedCallback([this](otChangedFlags aFlags) { HandleThreadStateChanged(aFlags); });
+    mIsEphemeralKeyEnabled = OTBR_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ON_INIT;
```

Comment:
nit: initialize this in the initialization list

Reply:
accept. thanks~